src/progly/run-papertests.cc

```diff
@@ -204,15 +127,15 @@ void do_sum( librados::bufferlist bl_seq, int write_type, bool debug ) {
 
   // perform sum
   uint64_t the_sum = 0 ;
-  for( int i = 0; i < these_ints.size(); i++ ) {
+  for( unsigned int i = 0; i < these_ints.size(); i++ ) {
     the_sum = the_sum + these_ints[i] ;
   }
   std::cout << "sum : " << std::to_string( the_sum ) << std::endl ;
   std::cout << "num ints : " << std::to_string( these_ints.size() ) << std::endl ;
 }
 
 void do_randomsearch( librados::bufferlist bl_seq, int write_type, bool debug ) {
-  int search_for = 551851 ;
+  uint32_t search_for = 4848 ;
```

Comment:
Are we certain this value 4848 will be present?  What happens if not found?

Reply:
I generated the csv data file once and picked the integer myself. It will be in there as long as the same generated data is used for the tested.

Reply:
ok, got it.